### ISSUE TITLE
GUI TODO LIST

### ISSUE THREAD

**Issue by alcomposer** [sfztools/sfizz#418]
## NEEDED:
* About sfizz pop-up when LMB on "sfizz" logo panel A
* Piano Keyboard widget with Keyswitch display
* CC Display with tab "pages"
* Navigation to other CC "pages"
* File tools in downward chevron in panel B
    * "load SFZ file"
    * "edit in external text editor"
    * "new"
    * "duplicate"
* Click on instrument name to open file open dialog
* Keyboard Widget UI setting:
    * Show/Hide `sw_down` when inside lokey/hikey range (default hide)
* Current keyswitch active label in panel B. LMB to choose from dropdown list
* Play Keyboard widget with LMB
* Show current MIDI notes in keyboard widget
* Pan & Volume rotary knob panel C
* Active tab dot or dash (panel A)
* Hover over tab rounded rectangle (panel A)

## NICE TO HAVE, BUT IMPORTANT
* File loading Throbber & Status Light- LMB on status icon opens log viewer popup
* Transposition & Tune control (panel B)
* Change Max Polyphony in panel B directly (downward chevron dropdown)
* Rotary knobs for CC's that match current look

## NICE TO HAVE:
* Display full path name for sample & labels in info tab (region count etc)
* Next/Previous SFZ script sideways arrows next to loaded file name (panel B)
* Histogram of all published CC's in CC tab
* Output Level Meter
* Pedals, pitchbend & modulation widgets in keyboard widget
* Current Memory footprint display (panel B)

**Comment by trebmuh** [comment]
I came here to ask for a visual feedback on the MIDI keyboard display, and I'm glad to see that there are much more on the way! 
Keep up the very good work all.
Thanks for sfizz. :+1:

**Comment by jpcima** [comment]
[midi_socket_icons.zip](https://github.com/sfztools/sfizz/files/6056201/midi_socket_icons.zip)

**Comment by SirGawain99** [comment]
Please provide an easy-to-find sfizz version number somewhere in the interface, or tell me where it's hidden. Sfizz is the best thing to happen to Linux music creation since Reaper. Finally, we may be free from the complexity of Sforzando + LinVST + wine etc.

**Comment by paulfd** [comment]
You can click on the sfizz logo.

**Comment by SirGawain99** [comment]
> You can click on the sfizz logo, on the top left.

Rats! It was that simple all along. Thank you. Perhaps, if the "sfizz" logo turned orange with a mouse-over, like the Info icon (where I expected to find this), it would be more obvious for dummies.

**Comment by paulfd** [comment]
Yeah you're right, it'd be a good indication. I'll add the version in the options panel too.

**Comment by paulfd** [comment]
> Rats! It was that simple all along. Thank you. Perhaps, if the "sfizz" logo turned orange with a mouse-over, like the Info icon (where I expected to find this), it would be more obvious for dummies.

From the next version you can also see the version in the option tab, and I added an about button in there.
